boat-scaffold/src/main/java/org/openapitools/codegen/languages/BoatSwift5Codegen.java

```diff
@@ -62,6 +69,49 @@ public String getTypeDeclaration(Schema p) {
         return super.getTypeDeclaration(p);
     }
 
+    @Override
+    public CodegenModel fromModel(String name, Schema model) {
+        Map<String, Schema> allDefinitions = ModelUtils.getSchemas(this.openAPI);
+        CodegenModel codegenModel = super.fromModel(name, model);
+        if (codegenModel.description != null) {
+            codegenModel.imports.add("ApiModel");
+        }
+
+        fixAllFreeFormObject(codegenModel);
+
+        return codegenModel;
+    }
+
+    private void fixAllFreeFormObject(CodegenModel codegenModel) {
```

Comment:
Would be good to add documentation to this method why this is needed. e.g. is it a workaround for a bug in the OpenApiClientGenerator, is it simply for backwards compatibility, what is the required behaviour vs current behaviour etc.